pkg/lib/authn/user/queries.go

```diff
@@ -181,3 +182,114 @@ func (p *Queries) GetMany(ids []string, role accesscontrol.Role) (users []*model
 
 	return
 }
+
+func (p *Queries) GetPageForExport(offset uint64) (users []*userexport.UserForExport, err error) {
```

Comment:
Move the type `userexport.UserForExport` to pkg/lib/authn/user/model.go

Reply:
moved to under user model